src/components/Extensibility/hooks/useDebugger.scss

```diff
@@ -0,0 +1,46 @@
+.debugonata {
+  width: auto;
+  max-height: 50vh !important;
+
+  h1 {
+    font-weight: bold;
+    border-bottom: 1px solid var(--sapTextColor);
+    padding: 0.5rem;
+  }
```

Comment:
all texts are very small and unreadable for me, could we adjust font size to the rest of busola?